recipes/python/assets/create_oracle_asset.py

```diff
@@ -0,0 +1,247 @@
+
+    if oracle_instance_id_uri != '':
+        print("\nOracle instance created.")
+
+    return oracle_db_id_uri
+
+def discover_oracle_databases(host):
+    discover_databases_request = open_sample_payload("post_oracle_discover.json")
+    discover_databases_request['data']['attributes']['parameters']['objectList'][0]['clientName'] = host
+    oracle_discover_response = requests.post(asset_service_url, headers=headers, json=discover_databases_request, verify=False)
+
+    if oracle_discover_response.status_code != 201:
+        print("\nAPI returned status code {}. Response: {}".format(oracle_create_db_response.status_code,
+                                                                 oracle_create_db_response.json()))
+        raise SystemExit("\nScript ended.\n")
+
+    print ("\nRequest to discover Oracle DBs on " + host +" has been posted.")
+
+def add_rman_catalog(catalogName, tnsName, credentialName):
+    rman_catalog_create_request = open_sample_payload("post_oracle_create_rman_catalog.json")
+    rman_catalog_create_request['data']['attributes']['parameters']['objectList'][0]['asset']['tnsName'] = tnsName
+    rman_catalog_create_request['data']['attributes']['parameters']['objectList'][0]['asset']['tnsName'] = rmanCatalogName
+    if credentialName != '':
+        rman_catalog_create_request['data']['attributes']['parameters']['objectList'][0]['asset']['commonAssetAttributes']['credentials']= [{"credentialName": credentialName}]
+    oracle_create_rman_catalog_response = requests.post(asset_service_url, headers=headers, json=rman_catalog_create_request, verify=False)
+
+    if oracle_create_rman_catalog_response.status_code != 201:
+        print("\nAPI returned status code {}. Response: {}".format(oracle_create_rman_catalog_response.status_code,
+                                                                 oracle_create_rman_catalog_response.json()))
+        raise SystemExit("\nScript ended.\n")
+
+    print ("\nRequest to create RMAN catalog has been posted.")
+
+    oracle_rman_catalog_uri = verify_asset_creation(oracle_create_rman_catalog_response.json()['data']['id'])
+    if oracle_rman_catalog_uri != '':
+        print("\nOracle RMAN Catalog created.")
+        return oracle_rman_catalog_uri.split('/')[-1]
+    else:
+        print("RMAN catalog creation did not respond in time, continuing without mapping the databse to the catalog")
+        return ''
+
+def add_credentials(credentialOption, credentialName):
+    print("Adding Credentials")
+    creds_create_request = open_sample_payload("post_oracle_add_creds.json")
+    creds_create_request['data']['attributes']['name']= credentialName
+    if credentialOption == "wallet":
+        creds_create_request['data']['attributes']['contents'] = {"useWallet": True}
+    elif credentialOption == "oracle" or credentialOption ==  "osAndOracle":
+        print("Enter Oracle Username")
+        oraUser = input()

```

Comment:
The `input` function accepts a prompt. Also include some separation after the prompt, or else the user's input will abut the prompt, making for a strange visual.

```suggestion
        oraUser = input("Enter Oracle Username: ")
```

Reply:
Changed all the inputs